src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/Cmdlets/UpdatePackageCommand.cs

```diff
@@ -150,6 +153,19 @@ protected override void ProcessRecordCore()
             TelemetryService.EmitTelemetryEvent(actionTelemetryEvent);
         }
 
+        protected override void ValidateArgumentsAreSupported()
+        {
+            
+            if (Source != null)
+            {
+                var projectNames = string.Join(",", Projects.Where(e => e.ProjectStyle == ProjectStyle.PackageReference).Select(p => NuGetProject.GetUniqueNameOrName(p)));
```

Comment:
Will this ProjectStyle be set for every case? or how about instead of ProjectStuyle, check for NuGetProject type being BuildIntegrated?

Reply:
It's always set. 
I personally prefer to use project style because it's easier to read?

What would the error message be then?
Do we mention both package reference and project json?